docs/release/procedure.md

```diff
@@ -0,0 +1,373 @@
+   `release/v1.2.x`. Ensure the tags are signed according to the [general
+   rules](#general-rules)., and push them to the remote repository.
+
+   ```shell
+   git switch release/v1.2.x
+   git pull origin release/v1.2.x
+   git tag -s -m "api/v1.2.1" api/v1.2.1
+   git push origin api/v1.2.1
+   git tag -s -m "v1.2.1" v1.2.1
+   git push origin v1.2.1
+   ```
+
+7. Confirm that the CI builds and releases the newly tagged version.
+
+8. Cherry-pick the changelog entry from the release series branch and create a
+   pull request to merge it into `main`.
+
+   ```shell
+   git pull origin main
+   git switch -c pick-changelog-v1.2.1 main
+   git cherry-pick -x <commit hash>
+   git push origin pick-changelog-v1.2.1
+   ```
+   
+#### Controller release candidates
+
+In some cases, it may be necessary to release a [release
+candidate](controllers.md#release-candidates) of a controller.
+
+To create a first release candidate, follow the steps to create a [minor
+release](#controller-minor-releases), but use the `rc.X` suffix for SemVer
+version to release (e.g., `v1.2.0-rc.1`).
+
+Once the release series branch is created, subsequent release candidates and
+the final (non-RC) release should follow the procedure for [patch controller
+releases](#controller-patch-releases).
+
+#### Controller preview releases
+
+In some cases, it may be necessary to release a preview of a controller.
+A preview release is a release that is not intended for production use, but
+rather to allow users to quickly test new features or an intended bug fix, and
+provide feedback.
+
+Preview releases are made by triggering the release GitHub Actions workflow on
+a specific Git branch. This workflow will build the container image, sign it
+using Cosign, and push it to the registry. But does not require a Git tag,
+GitHub release, or a changelog entry.
+
+To create a preview release, follow the steps below.
+
+1. Navigate to `https://github.com/fluxcd/<name>-controller/actions/workflows/release.yml`.
+2. Click the `Run workflow` button.
+3. Select the branch to release from the `Branch` dropdown.
+4. The default values for the `image tag` (`preview`) should be correct, but can
+   be changed if necessary.
+5. Click the green `Run workflow` button.
+6. The workflow will now run, and the container image will be pushed to the
+   registry. Once the workflow has completed, the image reference will be
+   available in the logs, and can be shared in the relevant issue or pull
+   request.
+
+### Distribution
+
+To release a [Flux distribution](flux.md) as a project maintainer, follow the
+steps below.
+
+Note that the Flux distribution contains multiple components, and you may need
+to release [controllers](#controllers) before releasing the distribution.
+Automation is in place to automatically create a pull request to update the
+version in the `main` branch when a new controller version is released.
+
+#### Distribution minor releases
```

Comment:
```suggestion
#### Distribution of minor releases
```
or
```suggestion
#### Distributing minor releases
```
maybe? (same goes for the below sections)

Reply:
They're "minor release of the (Flux) distribution". Maybe something like the following would be more clear?

> Distribution: minor releases

> Controller: minor releases

> etc.

The main reason to work with a prefix is to have unique refs for the titles (which can then be used in the ToC).

Reply:
yeah "Distribution: minor releases" seems good